src/api/spec/models/package_spec.rb

```diff
@@ -781,4 +781,34 @@
       it { expect { subject }.to raise_error(Package::CycleError) }
     end
   end
+
+  describe '#report_bug_url' do
+    # Locally the configuration returns https://unconfigured.openbuildservice.org
+    # as host so we use a better host
+    before { allow(Configuration).to receive(:obs_url).and_return('https://localhost:3000') }
+
+    context 'url is external' do
+      let(:package) { build(:package, report_bug_url: 'https://example.com') }
+
+      it { expect(package).to be_valid }
```

Comment:
How do you know why it is or isn't valid? Could be any validation...

Reply:
The package is build valid, and I only change the `report_bug_url` so if it's invalid it has to be this change

Reply:
Fixed